src/types/coin.ts

```diff
@@ -87,3 +87,36 @@ export interface CoinMPositionTrade {
   buyer: boolean;
   maker: boolean;
 }
+
+export interface FundingRateInfo {
+  symbol: string;
+  adjustedFundingRateCap: string;
+  adjustedFundingRateFloor: string;
+  fundingIntervalHours: number;
+}
+
+export interface IndexPriceConstituentsParams {
+  symbol: string;
+}
```

Comment:
Is this used anywhere? I don't see it being used. For such a simple interface, I would also prefer to just use `{ symbol: string }` as the type where it's needed, instead of having a separate interface.